fixes #4

Idrinth opened the pull request:
defines license to use as MIT

Idrinth committed:
fixes #4
defines license to use as MIT

Maurice-Betzel reviewed (changes requested):
Only where the dependencies are not lesser Apache 2.

Idrinth committed:
Update README.md

falling back on apache2 to make sure we still have the single license across all sub-projects
